js/territory/typeahead.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Typeahead.js/Bloodhound completion definition for territories
+ */
+define([
+    'jquery',
+    'api.light',
+    'bloodhound',
+    'templates/search/header.hbs',
+    'templates/search/suggestion.hbs',
+    'i18n',
+    'logger'
+], function($, API, Bloodhound, header, suggestion, i18n, log) {
+    var engine = new Bloodhound({
+            queryTokenizer: Bloodhound.tokenizers.whitespace,
+            datumTokenizer: function(d) {
+                return Bloodhound.tokenizers.whitespace(d.name);
+            },
+            identify: function(d) {
+                return d.id;
+            },
+            remote: {
+                url: API.build_url('/territory/suggest/') + '?q=%QUERY&size=',
```

Comment:
Actually, I'm only returning one result anyway so maybe I should just remove the parameter? I initially let it to be consistent with other calls.